app/subcommands/stop_subcommand.py

```diff
@@ -91,5 +91,9 @@ def _terminate_running_procs(self, procs_to_termintate):
 
     def _kill_running_procs(self, procs_to_kill):
         for proc in [p for p in procs_to_kill if p.is_running()]:
-            self._logger.info("Sending SIGKILL to PID {}", proc.pid)
-            os.kill(proc.pid, signal.SIGKILL)
+            if 'SIGKILL' in dir(signal):
+                self._logger.info("Sending SIGKILL to PID {}", proc.pid)
+                os.kill(proc.pid, signal.SIGKILL)
+            else:
+                self._logger.warning("Can't kill PID {} as SIGKILL is not supported on the platform. This is expected"
+                                     "if ClusterRunner is running on Windows.", proc.pid)
```

Comment:
Want to move this functionality into [`app/util/process_utils.py`](../../blob/master/app/util/process_utils.py)? I think there are a few places in the app where we might need this change. :)